tests/command_line/test_ssx_integrate.py

```diff
@@ -1,81 +1,90 @@
-from pathlib import Path
-
+import procrunner
 import pytest
 
 from dxtbx.serialize import load
 
 from dials.array_family import flex
-from dials.command_line.ssx_integrate import run as run_integrate
+from dials.command_line.ssx_integrate import run_integration, working_phil
+from dials.util.options import ArgumentParser
 
 # Note that tests are grouped and run serially, to stop many processes trying to
 # extract data from images at same time, which appears to lead to race
 # conditions in some CI jobs.
 
 
 @pytest.mark.xdist_group(name="group1")
-def test_ssx_integrate_stills(dials_data, run_in_tmpdir):
+def test_ssx_integrate_fullprocess(dials_data, tmp_path):
+    # Download data set and the internally referenced images
     ssx = dials_data("cunir_serial_processed", pathlib=True)
-    _ = dials_data("cunir_serial", pathlib=True)
-    indexed_refl = str(ssx / "indexed.refl")
-    indexed_expts = str(ssx / "indexed.expt")
-
-    run_integrate(
+    dials_data("cunir_serial", pathlib=True)
+    result = procrunner.run(
         [
-            indexed_refl,
-            indexed_expts,
-            "algorithm=stills",
+            "dev.dials.ssx_integrate",
+            str(ssx / "indexed.refl"),
+            str(ssx / "indexed.expt"),
```

Comment:
```suggestion
            ssx / "indexed.refl",
            ssx / "indexed.expt",
```
procrunner can deal with path objects